src/components/KymaModules/KymaModulesCreate.js

```diff
@@ -374,6 +489,15 @@ export default function KymaModulesCreate({ resource, ...props }) {
           className="collapsible-margins"
           title={t('kyma-modules.modules-channel')}
         >
+          {isSomeModuleUnmanaged && (
+            <MessageStrip
+              design="Information"
+              hideCloseButton
+              style={spacing.sapUiTinyMarginBottom}
+            >
+              {t('kyma-modules.unmanaged-modules-info')}
+            </MessageStrip>
+          )}
```

Comment:
Why don't we move this and `isSomeModuleUnmanaged` into a component, like `<UnmanagedModuleInfo kymaResource={kymaResource}/>`?